### DISCUSSION TITLE
Chrome 127 results

### DISCUSSION THREAD

**Discussion by krausest** [krausest/js-framework-benchmark#1720] [Announcements]
Results for chrome 127 are [published](https://krausest.github.io/js-framework-benchmark/2024/table_chrome_127.0.6533.72.html). Once again it looks a bit faster than chrome 126!

There a still a lot of alternative vanillajs versions in this run. Cleaning up is still on the TODO list.

We're using Chrome for Testing 127.0.6533.72 for this run.

Attached you find a binary build of all frameworks such that you can run the benchmark without installing and building all frameworks (see README for how to use the build.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/krausest/js-framework-benchmark/releases/tag/chrome127'>Chrome 127 results</a>.</em>

**Comment by leeoniya** [comment]
@krausest looks like `sonnet` is missing a https://github.com/krausest/js-framework-benchmark/issues/772 implementation note?
